### DISCUSSION TITLE
Where to configure the Notify Service

### DISCUSSION THREAD

**Discussion by quickclix75** [RogerSelwyn/O365-HomeAssistant#265] [Q&A] [answered]
I've added the MS365 Integration and setup the connection to my account, however I cannot figure out how to get the notify.o365_email_xxxxxxxx setup. Please can you let me know where I configure it so I am able to generate and send emails?

**Comment by RogerSelwyn** [comment]
Did you enable updates for the integration?

**Reply by quickclix75** [reply]
I believe so.. this is the integration config I've added

o365:
  accounts:
    - account_name: MicrosoftPrimaryPersonalAccount
      client_id: !secret office_365_client_id
      client_secret: !secret office_365_secret
      alt_auth_method: False
      enable_update: True
      enable_calendar: False
      groups: False
      track_new_calendar: False
      email_sensor:
        - name: inbox
          max_items: 0
          download_attachments: False

**Comment by RogerSelwyn** [comment, marked as answer]
I see that is the O365 integration. I’d have to check the code but there should be a service available to you. 

That said I suggest you use MS365_Mail integration - https://github.com/RogerSelwyn/MS365-Mail - O365 will be phased out over the next year.

**Reply by quickclix75** [reply]
Thanks @RogerSelwyn moved over to MS365-mail instead and works a treat!

**Comment by RogerSelwyn** [comment]
Looking at the code you just need enable update at the top level. You don’t need the inbox sensor for it to be made available.
